### PR TITLE
Cleanup secure-boot imx security rm pages

### PR DESCRIPTION
**Pull request by kprosise**
Minor changes for readability and style made to the two pages.

QA Steps: checked rendered output, no issues noticed. Ran linter.

This commit applies to FFTK-2795

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

File: source/reference-manual/security/secure-boot-imx-ahab.rst [hunk 1]
```diff
@@ -54,12 +54,13 @@ For development purposes, we keep i.MX AHAB sample keys and certificates at `lmp
...
-The Security Reference Manual for your specific SoC indicates which fuses need to be programed with the SRK fuse information.
+The Security Reference Manual for your specific SoC indicates which fuses need to be programmed with the SRK fuse information.
 
-i.MX 8QM fusing
---------------------------
+i.MX 8QM Fusing
+---------------
 
-.. note:: The values shown in this section are just examples of our standard LmP AHAB keys and are not meant for production. Fuses cannot be changed after the first write.
+.. note:: The values shown in this section are just examples of our standard LmP AHAB keys, and are not meant for production.
```
Comment (vanmaegima) [inline source/reference-manual/security/secure-boot-imx-ahab.rst:62]: Let's change this to a warning
```suggestion
.. note:: The values shown in this section are just examples of our standard LmP AHAB keys, and are not meant for production.
```

File: source/reference-manual/security/secure-boot-imx-ahab.rst [hunk 2]
```diff
@@ -54,12 +54,13 @@ For development purposes, we keep i.MX AHAB sample keys and certificates at `lmp
...
+   Fuses cannot be changed after the first write.
```
Comment (vanmaegima) [inline source/reference-manual/security/secure-boot-imx-ahab.rst:63]:
```suggestion
   Fuses cannot be changed after the first write.
```
is this being corrected compiled into the note block?

File: source/reference-manual/security/secure-boot-imx-ahab.rst [hunk 3]
```diff
@@ -132,24 +135,29 @@ Upon reboot, if **CONFIG_AHAB_BOOT** is set AHAB will raise events to indicate t
...
-This template is used by the ``lmp-tools/security/imx_ahab/sign-file.sh`` script which dynamically generates the authenticate data command "Offsets" line based on imx-boot image.  The command "Offset" line contains two values:
+This template is used by ``lmp-tools/security/imx_ahab/sign-file.sh`` which dynamically generates the authenticate data command "Offsets" line, based on imx-boot image.
+The "Offset" line contains two values:
 
-* Container header - offset to header of container, which contains set of binary images that should be signed
-* Signature block - offset to the signature block header
+* Container header: offset to header of container, which contains the set of binary images that should be signed
+* Signature block: offset to the signature block header.
 
 .. note::
```
Comment (vanmaegima) [inline source/reference-manual/security/secure-boot-imx-ahab.rst:156]:
```suggestion
.. note::
```
warning

File: source/reference-manual/security/secure-boot-imx-habv4.rst [hunk 4]
```diff
@@ -52,17 +64,20 @@ How to Secure the Platform
...
-The first step is to generate the PKI tree and commit the fuse table to the hardware.
+The first step is to generate the PKI tree, and commit the fuse table to the hardware.
 
  .. warning::
 
-   Once the fuses have been programmed they can't be modified.
+   Once the fuses have been programmed they can not be modified.
 
-Please refer to NXP's `Secure Boot Using HABv4 Guide`_ for a detailed description on how to generate the PKI tree.
+Please refer the NXP® `Secure Boot Using HABv4 Guide`_ for a detailed description on how to generate the PKI tree.
```
Comment (vanmaegima) [inline source/reference-manual/security/secure-boot-imx-habv4.rst:76]:
```suggestion
Please refer to the NXP® `Secure Boot Using HABv4 Guide`_ for a detailed description on how to generate the PKI tree.
```

File: source/reference-manual/security/secure-boot-imx-habv4.rst [hunk 5]
```diff
@@ -79,7 +94,8 @@ The Security Reference Manual for your specific SoC will indicate which fuses ne
 i.MX 7ULP Fusing
 ^^^^^^^^^^^^^^^^
 
-.. note:: The values shown in this section are just examples of our standard LmP HABv4 keys and are not meant for production. Fuses cannot be changed after the first write.
+.. note:: The values shown in this section are just examples of our standard LmP HABv4 keys, and are not meant for production.
+	 Fuses cannot be changed after the first write.
```
Comment (vanmaegima) [inline source/reference-manual/security/secure-boot-imx-habv4.rst:98]:
```suggestion
.. note:: The values shown in this section are just examples of our standard LmP HABv4 keys, and are not meant for production.
	 Fuses cannot be changed after the first write.
```
Change it to a warning
Check if the second line is being correctly added to the note

File: source/reference-manual/security/secure-boot-imx-habv4.rst [hunk 6]
```diff
@@ -150,7 +167,8 @@ And the following script would work for setting the M4 fuses::
 i.MX 8MM Fusing
 ^^^^^^^^^^^^^^^
 
-.. note:: The values shown in this section are just examples of our standard LmP HABv4 keys and are not meant for production. Fuses cannot be changed after the first write.
+.. note:: The values shown in this section are just examples of our standard LmP HABv4 keys and are not meant for production.
+	 Fuses cannot be changed after the first write.
```
Comment (vanmaegima) [inline source/reference-manual/security/secure-boot-imx-habv4.rst:171]:
```suggestion
.. note:: The values shown in this section are just examples of our standard LmP HABv4 keys and are not meant for production.
	 Fuses cannot be changed after the first write.
```
warning

**Comment by doanac** [comment]
Docs for 6f60a3fb4f8c809961a7adcdfcafca2b8bd8eee8 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2552/docs/artifacts/html/index.html

**Pull request merged**